tests/test_app/TestApp/Command/AutoLoadModelCommand.php

```diff
@@ -0,0 +1,9 @@
+<?php
+namespace TestApp\Command;
+
+use Cake\Console\Command;
+
+class AutoLoadModelCommand extends Command
+{
+    public $modelClass = 'Posts';
```

Comment:
Not in scope of this PR but this is on public property we seem to have forgotten to deal with.

Reply:
You want to make this protected for 4.0? :) Sure, sounds like a good idea.